data/genesis/decaf.toml

```diff
@@ -16,3 +18,17 @@ timestamp = "1970-01-01T00:00:00Z"
 
 [l1_finalized]
 number = 6758037 # Block where fee contract is deployed
+
+[[upgrade]]
+version = "0.3"
+start_proposing_view = 0
+stop_proposing_view = 4000000
+
+[upgrade.epoch]
+[upgrade.epoch.chain_config]
+chain_id = 0xdecaf
+base_fee = '1 wei'
+max_block_size = '1mb'
+fee_recipient = "0x0000000000000000000000000000000000000000"
```

Comment:
@jbearer : Should this be 0x00000... or should it be the prover address or something?

Reply:
0s is good